src/guide/receive_onchain.md

```diff
@@ -9,47 +9,60 @@ In such cases, the SDK can create a swap and receive the payment without the nee
 <section>
 
 ```rust,ignore
-{{#include ../../snippets/rust/src/receive_onchain.rs:generate-receive-onchain-address}}
+{{#include ../../snippets/rust/src/receive_payment.rs:prepare-receive-payment-onchain}}
+
+{{#include ../../snippets/rust/src/receive_payment.rs:receive-payment}}
```

Comment:
Looking at the rendered docs, I think it's confusing to see the parsing with different results. It should only be a BitcoinAddress. It's enough to highlight the `res.destination` like in the Swift example.
```
match parse(&res.destination).await {
    Ok(InputType::Bolt11 { .. }) => {}
    Ok(InputType::BitcoinAddress { .. }) => {}
    Ok(InputType::LiquidAddress { .. }) => {}
    _ => {
        dbg!("Unexpected destination: {}", res.destination);
    }
};
```